packages/contracts/src/viemClient.ts

```diff
@@ -117,6 +117,18 @@ export const viemClientByChainId: Record<ChainId, PublicClient> = {
   [KnownChainIds.BaseMainnet]: viemBaseClient,
 }
 
+export const viemNetworkIdByChainId: Record<ChainId, number> = {
```

Comment:
I've opted for the viem one because it is a direct inverse of the one below this, `viemClientByNetworkId`, rather than having to look for a home for a caip version of the same thing

Reply:
Absolutely fine by me, ty for rationale!